### PR TITLE
E2vid pip

### PR DESCRIPTION
**Pull request by danielgehrig18**
made e2vid pip installable, now you can cal e2vid with 

model = e2vid.E2VID(flags)
image = model(voxel_grid_numpy)

Internally model will still write the image to a folder, but you also have the possibility of 
reusing the image.

### CODE REVIEW

File: python/offline_reconstruction.py [hunk 1]
```diff
@@ -5,15 +5,14 @@
...
-from reconstruction.image_reconstructor import ImageReconstructor
-from reconstruction.options.inference_options import set_inference_options
-from reconstruction.utils.voxelgrid import VoxelGrid
+
+import e2vid
+from e2vid.options.inference_options import set_inference_options
+from e2vid.utils.voxelgrid import VoxelGrid
 
 
 def download_checkpoint(path_to_model):
```
Comment (Manasi94) [inline python/offline_reconstruction.py:18]: function duplicate (already defined and used here: python/reconstruction/e2vid/__init__.py)

File: python/reconstruction/e2vid/image_reconstructor.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
+import torch
+from e2vid.model.model import *
+from e2vid.utils.inference_utils import CropParameters, IntensityRescaler, ImageFilter, ImageWriter, UnsharpMaskFilter
+from e2vid.utils.event_tensor_utils import EventPreprocessor
+from e2vid.utils.image_display_utils import ImageDisplay
+from e2vid.utils.inference_utils import upsample_color_image, merge_channels_into_color_image  # for color reconstruction
+from e2vid.utils.timers import CudaTimer, cuda_timers
```
Comment (Manasi94) [inline python/reconstruction/e2vid/image_reconstructor.py:7]: since cuda time is no longer used, we can remove this import

**Comment by Manasi94** [comment]
lgtm

File: python/offline_reconstruction.py (hunk 1 above)
Comment (danielgehrig18) [reply python/offline_reconstruction.py]: fixed in latest commit

**Pull request merged**